docs/rules/no-unused-expressions.md

```diff
@@ -3,10 +3,10 @@
 Unused expressions are those expressions that evaluate to a value but are never used. For example:
 
 ```js
-"Hello world";
+someVar;
```

Comment:
What do you think about contrasting unused expression `n + 1;` which is a realistic typo of assignment  `n += 1;`